test/index.test.js

```diff
@@ -910,14 +910,33 @@ describe('Segment.io', function() {
           '{ "id": "xidxid" }'
         ]);
         server.respond();
-        
+
         var identify = segment.onidentify.args[0];
         analytics.assert(identify[0].traits().crossDomainId === 'xidxid');
-        
+
         analytics.assert(res.crossDomainId === 'xidxid');
         analytics.assert(res.fromDomain === 'userdata.example1.com');
         analytics.assert(!err);
       });
     });
   });
+
+  describe('when retryQueue is enabled', function() {
+    beforeEach(function(done) {
+      segment.options.retryQueue = true;
+      analytics.once('ready', done);
+      analytics.initialize();
+    });
+
+    afterEach(function() {
+      segment._lsqueue.stop();
+    });
+
+    it('#send should add to the retry queue', function() {
+      analytics.stub(segment._lsqueue, 'addItem');
```

Comment:
I can add a test that it ends up making the XHR request, which should test the full end-to-end behavior.